stable/jenkins/templates/config.yaml

```diff
@@ -36,7 +36,13 @@ data:
               <instanceCap>2147483647</instanceCap>
               <idleMinutes>0</idleMinutes>
               <label>{{ .Release.Name }}-{{ .Values.Agent.Component }}</label>
-              <nodeSelector></nodeSelector>
+              <nodeSelector>
+                {{- $local := dict "first" true }}
```

Comment:
I have never seen this before in gotpl. Can you explain what you are doing here?